### DISCUSSION TITLE
Each command for empty collection

### DISCUSSION THREAD

**Discussion by TimerkhanIskhakov** [jxlsteam/jxls#292] [Q&A] [answered]
I'm using jxls version 2.14.0 and when I get an empty collection in each command area, column under it shift up. How could I prevent such behavior? All I need is to keep column at it's place if I get an empty collection.

<img width="716" alt="image" src="https://github.com/jxlsteam/jxls/assets/82505420/2b825eed-b5aa-44d9-8ea3-6a511a54d185">
<img width="684" alt="image" src="https://github.com/jxlsteam/jxls/assets/82505420/108194cb-0ca2-4192-9353-79fe8aee7fab">

**Comment by SoltauFintel** [comment, marked as answer]
You could try to prepend a jx:if command before the jx:each command.

Please supply a Github repo with an example if you still have the problem.
